slugger.go

```diff
@@ -6,8 +6,9 @@ import (
 )
 
 // SlugifyTitle : Make sure to clean up the branch names
-func SlugifyTitle(ticketTitle string) string {
-	re := regexp.MustCompile("[^a-z0-9]+")
-
-	return strings.Trim(re.ReplaceAllString(strings.ToLower(ticketTitle), "-"), "-")
+func SlugifyTitle(ticketTitle string, replacementCharacter string) string {
+	re := regexp.MustCompile("[^a-z0-9]+");
+	lowerCaseTitle := strings.ToLower(ticketTitle)
+	formattedTitle := re.ReplaceAllString(lowerCaseTitle, replacementCharacter)
+	return strings.Trim(formattedTitle, replacementCharacter)
```

Comment:
Was a bit puzzled on how this was working, tried to make it a bit clearer by adding some steps